### ISSUE TITLE
Should images not be in the list?

### ISSUE THREAD

**Issue by neurosnap** [neurosnap/lists.sh#20]
I received some feedback that placing an image in the list is counter-intuitive.  My inclination is to limit the number of special features that "break out" of a list.  However, I did make the exception for headers so theoretically we could do the same for images.  My concern is that this is a microblog for lists and we should be trying to stay true to the list style rendering, but I'm open to alternative ideas.

**Comment by bentsai** [comment]
I haven't used images yet. And I'm primarily using this to blog, so a list of paragraphs with `list_type none`. So I'm sympathetic to making this act more like a traditional blogging engine. However, I'm still interested to see what novel behaviors prioritizing list-making might have on the idea and activity of microblogging. So I'm against having images break out of lists.

If you consider an image to represent a thing, then it makes sense for it to be in a list because you make lists of things:) If all the items in the list are images, then you have a special list that some might call a gallery.

**Comment by neurosnap** [comment]
I agree, a list of images is a gallery.  I'm going to close this issue because I don't think this issue stays true to the idea of lists.sh
